zebra/table_manager.c

```diff
@@ -130,22 +152,32 @@ struct table_manager_chunk *assign_table_chunk(uint8_t proto, uint16_t instance,
 	/* if not enough room space between MIN and COMPAT,
 	 * then begin after LOCAL
 	 */
-	if (start < RT_TABLE_ID_COMPAT && (size >
-				RT_TABLE_ID_COMPAT
-				- RT_TABLE_ID_UNRESERVED_MIN))
-		start = RT_TABLE_ID_LOCAL + 1;
+if (start < RT_TABLE_ID_COMPAT
```

Comment:
I think the style complaint is correct about this: you've left-shifted this block in this diff, but it looks like it should be inside the `if (!manual_conf) {` block, aligned with `tmc->start` at line 159 and 168? the associated comment should also be aligned inside the block opened by  `if (!manual_conf) {`.

Reply:
ok, I made an update by shifting the block inside `!manual_conf`.

Reply:
Hi @mjstapp , do you mind if you can finalise review pls?

Reply:
I think I'm done - we're just in the 8.1 hold window for another week?